cms/templates/js/course-outline.underscore

```diff
@@ -78,12 +78,14 @@ if (xblockInfo.get('graded')) {
                         </a>
                     </li>
                 <% } %>
-                <li class="action-item action-delete">
-                    <a href="#" data-tooltip="<%= gettext('Delete') %>" class="delete-button action-button">
-                        <i class="icon fa fa-trash-o"></i>
-                        <span class="sr action-button-text"><%= gettext('Delete') %></span>
-                    </a>
-                </li>
+                <% if (!(xblockInfo.isEntranceExam())) { %>
+                    <li class="action-item action-delete">
+                        <a href="#" data-tooltip="<%= gettext('Delete') %>" class="delete-button action-button">
+                            <i class="icon fa fa-trash-o"></i>
```

Comment:
I like to add an ```aria-hidden="true"``` to icon fonts as they can be focused on but don't speak anything by some screen readers.  They also render as a generic glyph in "visual" screen readers for people with cognitive disabilities. 
```<i class="icon fa fa-trash-o" aria-hidden="true"></i>```